utils/frame/try-runtime/cli/src/lib.rs

```diff
@@ -51,7 +51,7 @@ pub struct TryRuntimeCmd {
 	#[structopt(
 		long = "wasm-execution",
 		value_name = "METHOD",
-		possible_values = &WasmExecutionMethod::enabled_variants(),
+		possible_values = &WasmExecutionMethod::variants(),
 		case_insensitive = true,
 		default_value = "Interpreted"
```

Comment:
Is it intended that the default value is still interpreted here while in import-params it depends on the compilation flags?